### PR TITLE
Improve installation messages

### PR DESCRIPTION
**Pull request by binggh**
Closes: #19 

Printing before didn't show if failures happened while downloading, this should provide better visual guide on if `forc` or `fuel-core` failed to install.

### CODE REVIEW

File: src/commands/install.rs [hunk 1]
```diff
@@ -73,11 +93,18 @@ pub fn install() -> Result<()> {
...
+    if errored_bins_message.is_empty() {
+        info!("The Fuel toolchain is installed now. Great!");
+    } else if installed_bins_message.is_empty() {
+        error!("fuelup failed to install: {}", errored_bins_message)
+    } else {
+        info!(
+            "The Fuel toolchain is partially installed.\nfuelup failed to install: {}",
+            errored_bins_message
+        );
+    };
```
Comment (binggh) [inline src/commands/install.rs:107]: Are there better ways to print these messages? Clearer instructions to check for, etc.

File: src/commands/install.rs [hunk 2]
```diff
@@ -73,15 +93,21 @@ pub fn install() -> Result<()> {
...
+    if errored_bins_message.is_empty() {
+        info!(
+            "The Fuel toolchain is installed now. Great!\n You might need to add {} to your path.",
+            fuelup_bin_dir.display()
+        );
+    } else if installed_bins_message.is_empty() {
+        error!("fuelup failed to install: {}", errored_bins_message)
+    } else {
+        info!(
+            "The Fuel toolchain is partially installed.\nfuelup failed to install: {}\n You might need to add {} to your path.",
```
Comment (adlerjohn) [inline src/commands/install.rs:107]: Remove the adding to PATH sentence?

File: src/commands/install.rs [hunk 3]
```diff
@@ -73,15 +93,21 @@ pub fn install() -> Result<()> {
...
-    info!("\nThe Forc toolchain is installed now. Great!");
-    info!(
-        "\nYou might need to add {} to your path.",
-        fuelup_bin_dir.display()
-    );
+    info!("\nInstalled: {}\n", installed_bins_message);
+
+    if errored_bins_message.is_empty() {
+        info!(
+            "The Fuel toolchain is installed now. Great!\n You might need to add {} to your path.",
```
Comment (adlerjohn) [inline src/commands/install.rs:100]: Explain adding to path here.

File: src/commands/install.rs [hunk 4]
```diff
@@ -42,30 +42,55 @@ pub fn install() -> Result<()> {
...
+    let mut errored_bins_message = String::new();
+
+    match download_file_and_unpack(
         SWAY_RELEASE_DOWNLOAD_URL,
         &forc_release_latest_tag,
         &forc_bin_tarball_name,
-    )?;
+    ) {
+        Ok(()) => installed_bins_message.push_str(&format!("forc {}", &forc_release_latest_tag)),
+        Err(_) => errored_bins_message.push_str(&format!("forc {}", &forc_release_latest_tag)),
```
Comment (mitchmindtree) [inline src/commands/install.rs:55]: Another more concise/efficient way of writing to these strings is:

```rust
write!(&mut errored_bins_message, "forc {}", &forc_release_latest_tag);
```

Same with below

File: fuelup-init.sh [hunk 5]
```diff
@@ -115,12 +115,27 @@ main() {
...
 }
 
+post_setup_message() {
+    cat 1>&2 <<EOF
+
+fuelup v${_fuelup_version} has been installed in $FUELUP_DIR/bin. You might have to add $FUELUP_DIR/bin to path:
+
+bash/zsh:
+
+export PATH=\$HOME/.fuelup/bin:\$PATH
```
Comment (adlerjohn) [inline fuelup-init.sh:129]:
```suggestion
export PATH="\${HOME}/.fuelup/bin:\${PATH}"
```

Comment (adlerjohn) [reply fuelup-init.sh:129]: Outstanding comment ☝️

**Review by adlerjohn — APPROVED** [review]
utACK

**Pull request merged**